### DISCUSSION TITLE
Compiling the library in Windows.

### DISCUSSION THREAD

**Discussion by Elad1968** [Qalculate/libqalculate#597] [Q&A] [answered]
I want to compile the libqalculate library on windows to use it in a project, but I am not sure how to do so since "configure" is a bash script and cannot be run in Powershell. Is there a special environment to do it? Thank you in advanced!

**Comment by hanna-kn** [comment, marked as answer]
I have only compiled libqalculate using MSYS2/MinGW-w64.

**Reply by Elad1968** [reply]
Thanks a lot, got it working with MSYS2.
BTW, I got an error in the configure script for not having intltool installed, as it wasn't mentioned in the readme. If it's a dependency it probably should be mentioned in the requirements.
